content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,109 @@
+
+Nearly all the changes mentioned in this post were made to the _theme_ of our website which you can find [at this folder of the GitHub repo](https://github.com/ropensci/roweb2/tree/master/themes/ropensci/layouts). When modifying Hugo templates, [reading Hugo docs](https://gohugo.io/documentation/) is very helpful. When things go wrong (why doesn't this partial know about the author list?!) head to the [debugging template docs](https://gohugo.io/templates/template-debugging/). And if you can't figure out a way out, Hugo has got a [Discourse forum](https://discourse.gohugo.io/) like [all the cool kids these days](https://discuss.ropensci.org/). Needless to say, another useful tip when learning Hugo templating is to start small, with either a minimal theme and tiny website, or small updates, then your skills will improve.
+
+Other changes were adjustments to content such as YAML reformatting. Fear not, we rarely had to manually update metadata, we wrote scripts using the [`yaml` R package](https://CRAN.R-project.org/package=yaml). Find [such a script](https://github.com/ropensci/roweb2/issues/197#issuecomment-394264824).
+
+## More entry points to content
+
+On top of a few general pages, our website is the home of many [blog posts](/blog), [tech notes](/technotes) and [community calls](/commcalls). We've created and improved pages referencing content over all these categories, so that you might more easily bounce [from say one software peer review post to the next](/tags/software-peer-review), read [all posts by our community manager](/authors/stefanie-butland), or simply find [all the things we ever published](/archive).
+
+### Tags and authors pages
+
+We now have a [tags page](/tags) referencing all tags along with the number of posts associated to them. The tags of a blog post, tech note or comm call are shown at the bottom, so that after reading you can choose to check out more content on the topic. After tags became even more valuable, our community manager Stefanie Butland added many of them and for making that process smoother we've created a [Shiny app to update tags more easily over many posts, `blogyaml`](https://github.com/ropenscilabs/blogyaml). It's worked well for us, and if you notice something wrong, well it has an issue tracker!
+
+We also have added an [authors page](/authors) that's very similar. On each author's page, you'll find their title for rOpenSci staff members, their online home, possibly their Twitter handle, and all the content they've authored on our website. [See for instance Stefanie Butland's author page](authors/stefanie-butland). These pages are linked from the bottom of posts for returning authors. Thanks to [Angela Li](https://angela-li.github.io/) for [requesting we added authors pages](https://github.com/ropensci/roweb2/issues/365).
```

Comment:
Here it would be nice to link to a community member's author page, like `[See for instance Nick Tierney's author page](authors/nicholas-tierney/)`

Reply:
or to Kelly O'Briant or even both to feature different aspects